k8s-deploy/production/deployment_sidekiq.yaml

```diff
@@ -21,6 +21,7 @@ spec:
     spec:
       containers:
         - name: parliamentary-questions-rails-jobs
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/pq-team/parliamentary-questions:pq-main-e15d73ca
```

Comment:
Just wondering, was this change intended?

Reply:
nooo! will change it :)